src/profile-logic/import/instruments/BinaryPlistParser.js

```diff
@@ -0,0 +1,221 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
+
+export class UID {
+  index: number;
+
+  constructor(index: number) {
+    this.index = index;
+  }
+}
+
+class BinaryPlistParser {
+  view: DataView;
+  referenceSize: number;
+  offsetTable: Array<number>;
+
+  constructor(view: DataView) {
+    this.view = view;
+    this.referenceSize = 0;
+    this.offsetTable = [];
+  }
+
+  parseRoot(): any {
+    const trailer = this.view.byteLength - 32;
+    const offsetSize = this.view.getUint8(trailer + 6);
+    this.referenceSize = this.view.getUint8(trailer + 7);
+
+    // Just use the last 32-bits of these 64-bit big-endian values
+    const objectCount = this.view.getUint32(trailer + 12, false);
```

Comment:
it seems to be that all these `false` arguments to `getUint32` and friends aren't useful. Can you remove them all?